e2e/pages/enterChildren/enterChildren.js

```diff
@@ -45,43 +46,45 @@ module.exports = {
     activeChild = 'additionalChildren_0';
   },
 
-  enterChildDetails(name, day, month, year, gender = 'Boy') {
-    I.fillField(this.fields(activeChild).fullName, name);
-    I.fillField(this.fields(activeChild).DOB.day, day);
-    I.fillField(this.fields(activeChild).DOB.month, month);
-    I.fillField(this.fields(activeChild).DOB.year, year);
-    I.selectOption(this.fields(activeChild).gender, gender);
+  enterChildDetails(child) {
+    I.fillField(this.fields(activeChild).fullName, child.name);
+    I.fillField(this.fields(activeChild).DOB.day, child['DOB'].day);
```

Comment:
Shall we revert to `child.DOB.day`? Having `['DOB']` make code more difficult to refactor in the future.

In long term I would love to rename `DOB` to `dateOfBirth` as well.